src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FileTextOutlined } from '@ant-design/icons';
+import { Popover, Button } from 'antd';
+
+const OpenFillRequestFormButton = ({
+    props,
+  }) => (
+  props.config.features.exportToWorkspace?.openFillRequestForm && (
+    <Popover
+      className='discovery-popover'
+      arrowPointAtCenter
+      title={(
+        <React.Fragment>
+          &nbsp;
+          <a target='_blank' rel='noreferrer' href='https://pandemicresponsecommons.org'>
+            {'Pandemic Response Commons Website'}
```

Comment:
I hope we don't have to hardcode these env specific values in here
the url can be inferred from localconf, the text might be also about to be pulled from some value there
if not, maybe put them in configs

Reply:
I was planning to put it in config but I wasnt sure that it would be a good idea to just have config for value which will not change ever. I can add it into config if needed

Reply:
Put variable in the config, add external link icon

Reply:
external link icon in portal https://github.com/uc-cdis/data-portal/blob/master/src/components/layout/FooterNIAID.jsx#L162-L164

Reply:
external link icon and config variable for website and its name are added